### ISSUE TITLE
subctl diagnose firewall inter-cluster fails with Globalnet

### ISSUE THREAD

**Issue by skitt** [submariner-io/submariner-operator#1901]
I’m not sure about Globalnet being the reason, but it worked on the same clusters when Submariner was installed without Globalnet.

>  ✗ Checking if tunnels can be setup on the gateway node of cluster "skitt-1" 
>  ✗ The tcpdump output from the sniffer pod does not include the message sent from client pod. Please check that your firewall configuration allows UDP/4500 traffic on the "ip-10-0-150-48" node.

**Comment by sridhargaddam** [comment]
Did you use "subctl uninstall ...." before redeploying with Globalnet?

Also, can you please attach the diagnose output.

**Comment by skitt** [comment]
> Did you use "subctl uninstall ...." before redeploying with Globalnet?

Yes, I ran `subctl uninstall` on all the clusters involved, and it completed successfully.

> Also, can you please attach the diagnose output.

I no longer have it, but it didn’t find anything wrong. I have the `subctl gather` output if that’s any use. I can re-run the scenario to see if I reproduce the problem!

**Comment by sridhargaddam** [comment]
@skitt I'm moving it to generic backlog as we are not able to easily reproduce the issue.

**Comment by sridhargaddam** [comment]
In case we are unable to reproduce this issue, can we close it?

**Comment by skitt** [comment]
> In case we are unable to reproduce this issue, can we close it?

Yes, of course!
